### PR TITLE
Update heapwatch metrics

### PR DESCRIPTION
**Pull request by shdubsh**
Update heapwatch metric names and label usage per feedback in T238658

Bug: T238658

### CODE REVIEW

File: lib/heapwatch.js [hunk 1]
```diff
@@ -47,23 +47,19 @@ class HeapWatch {
...
                 Object.keys(this.cumulativeGCTimes).forEach((gcType) => {
                     const totalGCTime = this.cumulativeGCTimes[gcType];
                     if (totalGCTime > 0) {
-                        gcMetrics.timing(totalGCTime, [gcType]);
+                        this.metrics.makeMetric({
+                            type: 'Histogram',
+                            name: `gc.${gcType}`,
+                            prometheus: {
+                                name: `nodejs_gc_${gcType}_duration_ns`,
+                                help: 'garbage collection pause duration ns',
```
Comment (filippog) [inline lib/heapwatch.js]: We should use seconds as a base unit

File: lib/heapwatch.js [hunk 2]
```diff
@@ -47,23 +47,19 @@ class HeapWatch {
...
+                                staticLabels: { service: this.metrics.getServiceName() },
```
Comment (filippog) [inline lib/heapwatch.js]: AFAICT the GC and heap metrics are scoped to the whole nodejs runtime and not per-service (?) (internal to service runner), in this case we shouldn't attach `service` label. cc @Pchelolo to confirm/invalidate my assumption above

Comment (shdubsh) [reply lib/heapwatch.js]: The service name is populated by inspecting [package.json](https://github.com/wikimedia/service-runner/blob/prometheus_metrics/lib/base_service.js#L355) for the name field and defaulting to "service-runner."  From the metrics perspective, this name can be [overridden by adding a name field to the config.metrics[] member object.](https://github.com/wikimedia/service-runner/blob/prometheus_metrics/lib/base_service.js#L399)

Service name from the metrics perspective is populated [here](https://github.com/wikimedia/service-runner/blob/prometheus_metrics/lib/base_service.js#L383-L400).

Comment (ottomata) [reply lib/heapwatch.js]: Ya, and the service label is used to pull together most dashboards, so we should keep it.

**Pull request merged**